src/main/java/com/hubspot/jinjava/lib/tag/BlockTag.java

```diff
@@ -15,10 +15,7 @@
  **********************************************************************/
 package com.hubspot.jinjava.lib.tag;
 
-import com.hubspot.jinjava.doc.annotations.JinjavaDoc;
-import com.hubspot.jinjava.doc.annotations.JinjavaHasCodeBody;
-import com.hubspot.jinjava.doc.annotations.JinjavaParam;
-import com.hubspot.jinjava.doc.annotations.JinjavaSnippet;
+import com.hubspot.jinjava.doc.annotations.*;
```

Comment:
our style is to not use wildcard imports

Reply:
Fixed that, thanks for pointing that out